lib/sycamore/sycamore/tests/unit/transforms/test_standardizer.py

```diff
@@ -266,3 +267,41 @@ def test_ignore_errors_key_missing(self):
         key_path = ["nonExistentKey"]
         expected_output = {"event": {"coolKey": ""}}
         self.assertEqual(ignore_errors(doc, standardizer, key_path), expected_output)
+
+
+@pytest.mark.parametrize(
+    "raw, want",
+    [
+        ("March 17, 2023, 14.25 Local", "2023-03-17 14:25:00"),
+        ("March 17, 2023, 14.25", "2023-03-17 14:25:00"),
+        ("March 17, 2023 14:25:00", "2023-03-17 14:25:00"),
+        ("March 17, 2023 2:25PM", "2023-03-17 14:25:00"),
+        ("March 17, 2023 2:25AM", "2023-03-17 02:25:00"),
+        ("17 March 2023 14:25", "2023-03-17 14:25:00"),
+        ("2023-07-15 10.30.00", "2023-07-15 10:30:00"),
+        ("15/07/2023 10.30.00", "2023-07-15 10:30:00"),
+        ("2023-07-15 10.30.00 Local", "2023-07-15 10:30:00"),
+        ("2023-07-15 10.30.00PDT", "2023-07-15 10:30:00-07:00"),
```

Comment:
Why do we need to preserve the timezone? Why not convert to UTC time?

Reply:
It's not my decision to make.  I assume the reason is that sometimes the timezone isn't specified in either the document or the query and it's better to be in a position to do late binding.  The way datetime and dateparser work is that timezone can be specified if known and will be used if specified.  It's easy to convert a datetime to UTC if that's the goal.

Reply:
But if we want to 'standardize' it, isn't standardizing to one timezone part of the process? It seems like that's likely to reduce errors as well in that case. Also, how does OpenSearch do with timezones?